static/js/components/PersonalForm.js

```diff
@@ -105,28 +109,31 @@ export default class PersonalForm extends ProfileFormFields {
         </Cell>
         <Cell col={12}>
           <div className="section-header">
-            Where are you from?
+            Where are you from? <span
```

Comment:
It needs to be on the same line to have React include the space between the `?` and the `<`. If they are on separate lines React will assume there is no space between them